executor/executor.go

```diff
@@ -1175,6 +1175,7 @@ func ResetContextOfStmt(ctx sessionctx.Context, s ast.StmtNode) (err error) {
 	sc.MemTracker = memory.NewTracker(s.Text(), vars.MemQuotaQuery)
 	sc.NowTs = time.Time{}
 	sc.SysTs = time.Time{}
+	sc.PreparedParams = []interface{}{}
```

Comment:
Maybe put `PreparedParams` in `SessionVars` can avoid re-allocating memory before every execution.

Reply:
Why not set it to `nil`